### DISCUSSION TITLE
cors error wms geojson vector tile

### DISCUSSION THREAD

**Discussion by orttak** [kartoza/docker-geoserver#585] [Q&A]
I try to expalin in below post. If you have a time, could you check it? I also read Issues and discussion in this repo but I couldn't solve it
Thanks
https://gis.stackexchange.com/questions/469147/geoserver-cors-error-for-epsg3857-geojson/469198#469198

**Comment by orttak** [comment]
First, We need to open tomcat cors settings so we need to change web.xml under tomcat not GEOSERVER. 
If I use build compose file with "./build_data/web.xml:/settings/web.xml" setting, I can build image with cors open but when I use kartoza image from docker hub and apply "./build_data/web.xml:/settings/web.xml" setting when I create my container, cors settings do not apply it.

**Comment by NyakudyaA** [comment]
Does this issue still exists?

**Comment by NyakudyaA** [comment]
I use build compose file with "./build_data/web.xml:/settings/web.xml" setting - no need to do this as the image is build with this already, the only time you want to mount the web.xml is if you have custom change you need to apply with web.xml
